Makefile

```diff
@@ -1,6 +1,6 @@
 CC=$(HOME)/cheri/output/sdk/bin/riscv64-unknown-freebsd13-clang
-CFLAGS=-march=rv64imafdcxcheri -mabi=l64pc128d --sysroot=$(HOME)/cheri/output/rootfs-riscv64-hybrid -mno-relax -g -O0
-SSHPORT=10017
+CFLAGS=-march=rv64imafdcxcheri -mabi=l64pc128d --sysroot=$(HOME)/cheri/output/rootfs-riscv64-purecap -mno-relax -g -O0
```

Comment:
Can all examples be built in `purecap` mode?

Reply:
Yes they do.  Buildbot also supports purecap mode so this shouldn't be a problem.

Reply:
OK, so dumb question: why did we specify `hybrid` before?

Reply:
Because we/I didn't know any better, and somewhere it was written that purecap is broken so the hybrid one should be used.

Reply:
Point taken!